pkg/test/utils/test_context.go

```diff
@@ -32,8 +32,8 @@ const prometheusOperatorVersion = "0.33"
 const certmanagerURL = "https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml"
 const prometheusOperatorURL = "https://raw.githubusercontent.com/coreos/prometheus-operator/release-%s/bundle.yaml"
 
-// KBTestContext specified to run e2e tests
-type KBTestContext struct {
+// TestContext specified to run e2e tests\
```

Comment:
Nit: Typo on the trailing `\`
```suggestion
// TestContext specified to run e2e tests
```